front/app/containers/Admin/sideBar/MenuItem.tsx

```diff
@@ -96,27 +104,58 @@ const MenuItem = ({ navItem }: Props) => {
     item: { type: 'route', path: navItem.link },
   });
 
+  // Temporary proposal warning implementation, will be removed together with the navbar item
+  // after users have had enough time to get used to the feature
+
+  const isProjectProposalsEnabled = useFeatureFlag({
+    name: 'proposals_participation_method',
+  });
+
+  const isItemDisabled =
+    isProjectProposalsEnabled && navItem.name === 'initiatives';
+
   if (!featuresEnabled || !hasPermission) return null;
 
   return (
-    <MenuItemLink
-      to={navItem.link}
-      className={`intercom-admin-menu-item-${navItem.name}`}
+    <Tooltip
+      content={
+        <Box
+          display="flex"
+          flexDirection="column"
+          alignItems="center"
+          gap="20px"
+        >
+          <Image src={tooltipImage} alt="" w="250px" />
+          <FormattedMessage {...messages.proposalsTooltip} />
+        </Box>
+      }
+      placement="right"
+      disabled={!isItemDisabled}
+      theme="dark"
     >
-      <Box
-        display="flex"
-        flex="0 0 auto"
-        alignItems="center"
-        justifyContent="center"
-        className={navItem.iconName}
+      <MenuItemLink
```

Comment:
I fixed some accessibility issues for Dublin in another PR, and we had to remove the Box from within the Tooltip component (which wrapped the children), since it caused accessibility issues (`aria-expanded` was being set on the wrapper div instead of the button/input child element).

In order to use a Tooltip on a link like this, we just need to wrap it with a <div> or <Box> as well.

Reply:
Hm.. this seems a bit of an issue if every time you want to use the tooltip, you need to add a box also. Or am I misunderstanding the issue? Could you link to the PR with the fix you are referring to?

Reply:
Sure! It's [this one](https://github.com/CitizenLabDotCo/citizenlab/pull/8992) :)

Dublin flagged the issue where the `aria-expanded` attribute was being applied to spans/divs where it shouldn't be (`aria-expanded` can only be applied to [these roles](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded#associated_roles)), which happened because we were always wrapping the children in a div in our Tooltip component.

Maybe another solution could be a prop to specify whether or not we want to wrap the children in a div or something? :thinking:

Reply:
I think if this were a standard link element there wouldn't be an issue, but the RouterLink won't work without being wrapped in a div :thinking:

Reply:
I am a little afraid that this change will have consequences on many layouts that use tooltips and if the issue is only about tooltips that wrap expandable components I would go for introducing the prop. Otherwise we need to check and test many places on the platform to make sure it all still works as expected.

I don't fully remember anymore why but I know I added the Box for a good reason (initially the tooltip component was without it). If we decide to remove it, the whole tooltip behaviour should also be retested - like keyboard accessibility, appearing on focus, closing with `esc` key, etc. Also, as I mentioned, some layouts might be relying on the box being there.

Reply:
Ok clear - I'll go with the prop approach then! :)